src/Components/Forms/src/ValidationMessageStore.cs

```diff
@@ -65,7 +64,7 @@ public void Add(Expression<Func<object>> accessor, IEnumerable<string> messages)
         /// <param name="fieldIdentifier">The identifier for the field.</param>
         /// <returns>The validation messages for the specified field within this <see cref="ValidationMessageStore"/>.</returns>
         public IEnumerable<string> this[FieldIdentifier fieldIdentifier]
-            => _messages.TryGetValue(fieldIdentifier, out var messages) ? messages : Enumerable.Empty<string>();
+            => _messages.TryGetValue(fieldIdentifier, out var messages) ? messages : Array.Empty<string>();
```

Comment:
Just for my education, could you fill me in on why this makes a difference? Since the return type is `IEnumerable<string>` either way, I would have thought that `Enumerable.Empty<T>()` would already have returned the most performant thing that can satisfy that contract.

Reply:
Oh never mind, I realise it's probably just that you're removing the `System.Linq` reference.